src/fileWriters/index.js

```diff
@@ -1,12 +1,50 @@
-import path from 'path';
+import extractZip from 'extract-zip';
+import chalk from 'chalk';
+import axios from 'axios';
+import fse from 'fs-extra';
+import rimraf from 'rimraf';
 
 import writeTemplateDirectoryFiles from './writeTemplateDirectoryFiles';
+import createTemplateFiles from './createTemplateFiles';
+import {
+  SOURCE_URL,
+  SOURCE_FILENAME,
+  TEMP_DIR,
+  TEMPLATES_DIR,
+} from '../constants';
 
-const writeBaseTemplates = async ({ destination, templateValues }) =>
+export const writeBaseTemplates = async ({ destination, templateValues }) =>
   writeTemplateDirectoryFiles({
     destination,
     templateValues,
-    location: path.join(__dirname, '../templates'),
+    location: TEMPLATES_DIR,
   });
 
-export default writeBaseTemplates;
+const extractSource = async (source) => {
+  extractZip(source, { dir: TEMP_DIR }, async (error) => {
+    if (error) {
+      console.log(chalk.bold.redBright(error));
+      return;
+    }
+    await createTemplateFiles();
+  });
+};
+
+export const getCookieCutterSource = async () => {
```

Comment:
Should a method named `getCookieCutterSource` be in a file called `fileWriters`?

I'd expect methods named `writeXtoY` (or something like that) to be in `fileWriters`. I guess what I'm hinting at is that it seems that this method is doing a decent amount of things that are not just file writing.